### ISSUE TITLE
languages.xml is missing in some translations

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#4944]
**Describe the bug**
The languages metadata field offers autosuggest options when typing. These options always show English-language options, even when entering for another language.

After adjusting the request to fetch locale-specific suggestions, it became apparent that the languages.xml file is missing in some translations. For these locales, no autosuggest options are available. 

This also causes errors to appear in the server logs:

```
[24-Jul-2019 16:28:11 Europe/London] PHP Warning:  filemtime(): stat failed for lib/pkp/locale/fr_CA/languages.xml in /home/nate/Projects/pkp/ojs/lib/pkp/classes/language/LanguageDAO.inc.php on line 38
[24-Jul-2019 16:28:11 Europe/London] PHP Warning:  fopen(lib/pkp/locale/fr_CA/languages.xml): failed to open stream: No such file or directory in /home/nate/Projects/pkp/ojs/lib/pkp/classes/file/FileWrapper.inc.php on line 70
```

**To Reproduce**
1. Enable `en_US` and `fr_CA` languages.
2. Make a submission up to the metadata section.
3. Enter a letter into the english language field. You will see English-language suggestions appear.
4. Enter a letter into the french language field. You will see the same English-language suggestions, not translated to French.

**What application are you using?**
OJS 3.1.2 and `master` (pre 3.2)

**Comment by NateWr** [comment]
@mtub This file seems to be missing for some languages that are otherwise very complete (like `fr_CA`). Is it possible this is just getting overlooked during the translation process?

**Comment by asmecher** [comment]
How to take the Debian JSON data and generate the English list (using similar instructions to https://github.com/pkp/pkp-lib/issues/2715):

```
php -r '$a = json_decode(file_get_contents("iso-codes/json/iso_639-2.json"));foreach ($a as $b) foreach ($b as $c) {if (!isset($c->alpha_2)) continue; print "\t<language code=\"{$c->alpha_2}\" name=\"{$c->name}\"/>\n";}'
```
This will dump `<language ...>` elements out to the console, which can be dropped into `lib/pkp/locale/en_US/languages.xml` in place of the current set.

Then, to generate other languages, use the newly-added `lib/pkp/tools/poToLanguages.xml`.

PR to add the tool and update the translations:
https://github.com/pkp/pkp-lib/pull/4973

@mtub, I've assigned the review to you -- I note that the French PO files from Debian have translations all in lower case, but that seems like an upstream problem to me.

@marcbria, I didn't include a Spanish update because you already seem to have sourced a good version of the standard. Do you know where you got this from?

**Comment by marcbria** [comment]
@asmecher in spanish and catalan versions I translated the missing files "file by file" from the english version so may be this is why it's ok.
Or may be kudos need to go for @mtub that toke our all xmls files and probably fixed them before pushing to github. (Marco, I will be always in debt for this. ;-))

**Comment by NateWr** [comment]
Hah, brilliant Alec! I've chucked this into our versioning project because the issue is exposed with the new metadata forms there.

@mtub I'll be very grateful if you get a chance to approve this in the next month or so. If not, I may have to go forward with it anyway and we can clean up afterwards. We just need to make sure we're shipping some fix here with 3.2.

**Comment by asmecher** [comment]
Merged and cherry-picked forward to `master`!
